index2.html

```diff
@@ -0,0 +1,253 @@
+                ></path>
+              </g>
+            </svg>
+            <span class="footer-copyright align-self-end">
+              © 2021 NEWSWEEK DIGITAL LLC
+            </span>
+          </div>
+
+          <ul class="navbar-nav d-flex flex-row p-1 footer-links">
+            <li class="nav-item font-weight-bold">
+              <a class="nav-link text-decoration-none" href="#">Editions:</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">U.S. Edition</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">日本</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Pakistan</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Polska</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">România</a>
+            </li>
+          </ul>
+          <ul class="navbar-nav d-flex flex-row footer-links">
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">About Us</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Corrections</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Contact Us</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Editorial Guidelines</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Advertise</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Copyright</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Terms & Conditions</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Privacy Policy</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Cookie Policy</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Terms of Sale</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Archive</a>
+            </li>
+            <li class="nav-item border-right">
+              <a class="nav-link" href="#">Announcements</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </footer>
+
+    <script src="assets/jquery-3.5.1.min.js"></script>
+    <script src="assets/bootstrap/js/bootstrap.min.js"></script>
```

Comment:
- [ ] Duplication of code. Decide to go local or CDN.

Reply:
Done